buildspecs/al-maintenance-release-config.yml

```diff
@@ -0,0 +1,47 @@
+version: 0.2
+
+phases:
+  build:
+    commands:
+      - GITHUB_DEST_BRANCH_NAME=$(echo ${CODEBUILD_WEBHOOK_BASE_REF} | cut -d "/" -f3-)
```

Comment:
I think what Amogh is asking is if we can put all this in a shell script and then just run the script in spec file rather than inline all of these commands in the spec file. The only thing to note about that is that Codebuild runs all of the commands in their own `sh` instances (conceptually) and the shell script invocation would be all the commands in one instance. 

So if that works for us, we can make the change.

Reply:
Oh I see. Yeah that's something we can for sure test out.

Reply:
Just tested this out in my forked repo and personal dev account. It seems to be working with the changes I made.

Reply:
Thanks @mye956 ! Easier to read now!